src/main/java/com/untamedears/jukealert/model/appender/LeverToggleAppender.java

```diff
@@ -59,7 +59,9 @@ public boolean shouldToggle() {
 	
 	public void switchState() {
 		shouldToggle = !shouldToggle;
-		snitch.setDirty();
+		if (snitch.getId() != -1) {
+			JukeAlert.getInstance().getDAO().setToggleLever(snitch.getId(), shouldToggle);
```

Comment:
This feels like an odd choice. Unless I'm mistaken, wouldn't this mean that lever state changes would immediately commit to the DB? That'd be back to a user action synchronously blocking against the db update, and that's not ideal.

Although granted that I didn't see any other code committing the state, I'd still recommend pursuing making sure setToggleLever gets called in the dirty handler vs. a direct insert on the main thread?